### DISCUSSION TITLE
How to roll characters or 0 padded numbers?

### DISCUSSION THREAD

**Discussion by elibroftw** [barvian/number-flow#50] [Help]
Right now to roll 0-padded numbers or custom text (" - " between numbers) I have to wrap each digit with NumberFlow and then set the trend to increasing. I'm wondering if there's a way to roll non-digits (in case accountIds start allowing characters) and if there's a way to replace 20 numberflow components with just 5 which can handle 0 padded numbers. 

```jsx
function AccountId({ accountId }) {
  // every 4 digits, add a -
  let result = [];
  for (let i = 0; i < accountId.length; i += 1) {
    // result.push();
    result.push(<NumberFlow spinTiming={{ duration: 600, easing: 'ease-out' }} value={accountId.charAt(i)} trend='increasing' />)
    if (i % 4 === 3 && i !== accountId.length - 1) {
      result.push(' - ');
    }
  }
  return (
    <Card radius='md' withBorder w={300}>
      <Center><span>{result}</span></Center>
    </Card>
  );
}
```

**Comment by barvian** [comment]
Hi 👋, have you seen the [`minimumIntegerDigits` option](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#minimumintegerdigits)? I think that would help the padding part at least. [Sandbox](https://codesandbox.io/p/sandbox/numberflow-padded-frqdxv)
